### DISCUSSION TITLE
Android Auto

### DISCUSSION THREAD

**Discussion by Biker729** [devemux86/cruiser#510] [Q&A] [answered]
Hello, is it possible to use Cruiser with Android Auto and offline maps (BRouter). ? Greetings and thanks
Hallo, kann man Cruiser mit Android Auto und Offline Karten (BRouter) verwenden. ? Gruß und Danke

**Comment by devemux86** [comment, marked as answer]
Hi @Biker729,

Currently this is not possible. We also have a discussion about this here:

- https://github.com/devemux86/cruiser/discussions/508

**Reply by DIrgi** [reply]
Didn't I say it just today? 😉
